dev/sparktestsupport/modules.py

```diff
@@ -178,7 +178,7 @@ def __hash__(self):
 
 core = Module(
     name="core",
-    dependencies=[kvstore, network_common, network_shuffle, unsafe, launcher],
+    dependencies=[kvstore, network_common, network_shuffle, unsafe, launcher, utils],
```

Comment:
cc @zhengruifeng @HyukjinKwon ,  `utils` module is also a direct dependency of unsafe and network-common. Do we need to modify the dependencies of `unsafe` and `network-common`? Or is the current change sufficient?

Reply:
> utils module is also a direct dependency of unsafe and network-common

let's also add this dependency

Reply:
done